lib/config/routes/router.dart

```diff
@@ -1,4 +1,6 @@
 class MyRouter {
```

Comment:
As a convention all the routes should start with '/' and also the app home page should be '/' only, all the other routes should be named with lower snake case
Ex: 
`static const String movieView = '/movie_view'`

Reply:
Done